testkit/src/main/scala/org/bitcoins/testkit/wallet/BitcoinSWalletTest.scala

```diff
@@ -439,6 +440,7 @@ object BitcoinSWalletTest extends WalletLogger {
       wallet <- BitcoinSWalletTest.createWallet2Accounts(bitcoind,
                                                          bitcoind,
                                                          extraConfig)
+      _ = wallet.stopWalletThread()
```

Comment:
Why is this here? Shouldn't this be in `destroyWallet`?

Reply:
We create the wallet again so it can have callbacks after this, so this wallet will never passed to destroy wallet

Reply:
Ugh, so it seems this thread management should really happen in `WalletAppConfig` as well then?